flink-connector-kafka/src/test/java/org/apache/flink/KafkaAssertjConfiguration.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink;
+
+import org.assertj.core.configuration.Configuration;
+
+/** Configuration for AssertJ that shows full stack traces for unmatched exceptions. */
+public class KafkaAssertjConfiguration extends Configuration {
+    public KafkaAssertjConfiguration() {
+        setMaxStackTraceElementsDisplayed(10000);
```

Comment:
Since we recently reduced the log sizes significantly, what's the impact of this change?

Reply:
If you get a test failure, the log size is larger now. But you actually see the issue. For green builds, everything is unchanged.